### DISCUSSION TITLE
When will firebase auth becomes available?

### DISCUSSION THREAD

**Discussion by IskandarAlex2** [requarks/wiki#6760] [Help / Questions] [answered]
I wanted to use Wiki.js for making wiki for my indie game franchise, don't want to use fandom because the outrageous ads and will be too complex to be on wiki.gg. Besides, I already have a GCP project and a domain name so why not host one myself on Cloud Run. We used firebase to simply our workflow and we use Google as our authentication provider which I believe still count as Firebase authentication, If otherwise please let me know. I want the wiki I want to host uses firebase auth to make it compatible with our website which we make with Vue and using firebase services, but I realized it is still under development so I would like to know when will it be released, if it will be too long then I will just use Google auth directly.

I am sorry if my English is messed up because I am off to vacation and wrote this in my phone.

**Comment by NGPixel** [comment, marked as answer]
Unless someone submits a PR to add the firebase module, it won't be anytime soon. This is not a priority right now.
